src/GameClient.ts

```diff
@@ -73,6 +73,16 @@ export class GameClient extends Client {
     }
 
     /**
+     * Instructs the server to create a new scene with the specified parameters.
+     */
+    public createScene(scene: ISceneData): Promise<ISceneData> {
+        return this.execute('createScenes', { scenes: [ scene ] } as ISceneDataArray, false)
```

Comment:
How about,

`this.createScenes({scenes:[scene]})`

That way we don't have a duplicate `execute` call?